02/app.js

```diff
@@ -9,12 +9,16 @@ class App extends React.Component {
         usersList: [],
     }
 
+    addUserName = (user) => {
+        this.setState(prevState=>({usersList:[...prevState.usersList,user]}))
```

Comment:
Jak już to w tym miejscu generujemy uuid tj. `{usersList:[...prevState.usersList,{name: user, uuid: uuidv4()}]}`